.travis.yml

```diff
@@ -1,19 +1,26 @@
 language: python
+python:
+  - 3.6
 install:
   - pip install tox
 script: tox
 sudo: false
 env:
   matrix:
     - TOXENV=py27-django18
-    - TOXENV=py27-django19
     - TOXENV=py27-django110
+    - TOXENV=py27-django111
     - TOXENV=py33-django18
     - TOXENV=py34-django18
-    - TOXENV=py34-django19
+    - TOXENV=py34-django110
+    - TOXENV=py34-django111
     - TOXENV=py35-django18
-    - TOXENV=py35-django19
     - TOXENV=py35-django110
+    - TOXENV=py35-django111
+    - TOXENV=py36-django111
+    - TOXENV=pypy-django18
+    - TOXENV=pypy-django110
+    - TOXENV=pypy-django111
```

Comment:
I'm hesitant to adding pypy here. The build matrix is already quite large, but mainly because pypy is not officially supported by Django.
https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django